packages/wallet-lib/src/utils/castStorageItemsTypes.js

```diff
@@ -0,0 +1,54 @@
+const castStorageItemsTypes = (originalItem, schema) => {
```

Comment:
Why did you use Ajv? It has schema validation and type casting.

Reply:
Thanks for the info. This function has been written about 4 months ago, and we just transferred it there with slight modifications. I was not aware of the Ajv library back then. We shall consider replacing.